### PR TITLE
DX-2690, DX-2693 webrtc_api Integration Tests

### PR DESCRIPTION
**Pull request by ckoegel**

### CODE REVIEW

File: test/integration/test_webrtc_api.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+from bandwidth.model.create_participant_response import CreateParticipantResponse
+from bandwidth.model.publish_permissions_enum import PublishPermissionsEnum
+from bandwidth.model.device_api_version_enum import DeviceApiVersionEnum
+from bandwidth.model.participant import Participant
+from bandwidth.model.session import Session
+from bandwidth.model.subscriptions import Subscriptions
+from bandwidth.model.participant_subscription import ParticipantSubscription
+
+
+class TestSessionsApi(unittest.TestCase):
```
Comment (ajrice6713) [inline test/integration/test_webrtc_api.py:31]: Can you add docstrings to all of your functions?

Comment (ajrice6713) [inline test/integration/test_webrtc_api.py:31]: Does this API return proper 400/401/403 errors? In the other tests we created unauthorized and forbidden clients to test the 4xx responses - we should test these 4xx responses too

**Review by ajrice6713 — CHANGES_REQUESTED** [review]
LGTM other than the couple comments

**Comment by ajrice6713** [comment]
👀 Merge Conflict @ckoegel 👀

File: test/integration/test_webrtc_api.py (hunk 1 above)
Comment (ckoegel) [reply test/integration/test_webrtc_api.py:32]: done

Comment (ckoegel) [reply test/integration/test_webrtc_api.py:32]: There were already tests in there for a 401 and 404 error, but I couldn't find a way to force a 403 or 400. Most stuff in the api is optional and gets ignored if formatted incorrectly, and if you do something horrendously wrong, you get a 500. If you know how to force either of those lmk and I'll add the test.

Comment (ajrice6713) [reply test/integration/test_webrtc_api.py:32]: I dont know how to force those - just wanted to make sure it wasnt overlooked - but sounds like youve covered the bases we have available.

**Review by ajrice6713 — APPROVED** [review]

File: test/integration/test_webrtc_api.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    """SessionsApi unit test stubs"""
+
+    def setUp(self):
+        # API Client
+        configuration = bandwidth.Configuration(
+            username = os.environ.get('BW_USERNAME'),
```
Comment (brianluisgomez) [inline test/integration/test_webrtc_api.py:38]: For the environment variables, you could merge in the feature branch and it should add in a util file with all of those variables. Then you could just import and use them here without having to do the whole os.environ thing or having to import os at all. This should make these match the other test files for consistency.

File: test/integration/test_webrtc_api.py [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+        # Session Request
+        self.session = Session(
+            tag=self.session_tag
+        )
+
+        self.stream_aliases = ['python integration alias']
+
+    def create_participant(self):
+        """Test creating participant
```
Comment (brianluisgomez) [inline test/integration/test_webrtc_api.py:76]: I think most of these docstrings could be removed altogether. The individual test name is pretty self-explanatory

Comment (ajrice6713) [reply test/integration/test_webrtc_api.py:76]: lol my previous comment was to add the docstrings 

lets keep them in there for consistencies sake - IMO its expected

Comment (brianluisgomez) [reply test/integration/test_webrtc_api.py:76]: haha sounds good to me; it doesn't really make much difference either way

**Review by brianluisgomez — APPROVED** [review]

**Pull request merged**